src/Formatter/DateFormatter.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#include "DateFormatter.hpp"
+#include "time/BrokenDate.hpp"
+#include "Math/Util.hpp"
+#include "util/StringCompare.hxx"
+#include "util/StaticString.hxx"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+static constexpr const TCHAR* short_month[] = {
+  _T("Jan"),
+  _T("Feb"),
+  _T("Mar"),
+  _T("Apr"),
+  _T("May"),
+  _T("Jun"),
+  _T("Jul"),
+  _T("Aug"),
+  _T("Sep"),
+  _T("Oct"),
+  _T("Nov"),
+  _T("Dec")
+};
+
+static constexpr const TCHAR* long_month[] = {
+  _T("January"),
+  _T("February"),
+  _T("March"),
+  _T("April"),
+  _T("May"),
+  _T("June"),
+  _T("July"),
+  _T("August"),
+  _T("September"),
+  _T("October"),
+  _T("November"),
+  _T("December")
+};
```

Comment:
The short names should not be localized, the long names I will remove because not used up to now.
Which solution you are preferring for such static TCHAR* array?

Reply:
I prefer using ISO8601 until a proper solution is found.